index.html

```diff
@@ -0,0 +1,118 @@
+        <section 
+        class="main-program-section">
+            <div class="main-program-div">
+                 <h2>Main Program</h2>
+                <div class="main-program-item-container">
+                    <div class="main-program-item">
+                     
+                        <h3>Lecture</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                     
+                        <h3>CC Exhibition</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                        
+                        <h3>Forum</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                        
+                        <h3>Workshop</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                    <div class="main-program-item">
+                       
+                        <h3>CC Ignite</h3>
+                        <span>Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem, dolorum.</span>
+                    </div>
+                </div>
+            </div>
+        </section>
+                    <!-- feature speaker section -->
+        <section class="feature-speaker-section">
+        </section>
+    </main>
+        <!-- Footer Section -->
+        <footer>
+            <section class="partners">
+                <h3>Partners</h3>
+                <div class="partners-div">
+                    <ul class="partners-div-ul">
+                            <li class="partners-div-li"><img src="./assets/imgs/mozilla.png" alt="mozilla"></li>
+                            <li class="partners-div-li"><img src="./assets/imgs/google.png" alt="google"></li>
+                            <li class="partners-div-li"><img src="./assets/imgs/naver.png" alt="naver"></li>
+                            <li class="partners-div-li"><img src="./assets/imgs/daunkamao.png" alt="daunkako"></li>
+                            <li class="partners-div-li"><img src="./assets/imgs/airbnb.png" alt="airbnb"></li>
+                    </ul>
+                </div>
+            </section>
+            <section class="copy-right">
+                    <div class="copy-right-logo">
+                        <img src="./assets/imgs/logo.png" alt="logo">
+                    </div>
+                    <div class="copy-right-text">
+                        <span>2023 web dev some Rights Reserved &copy;</span>
+                    </div>
+            </section>
```

Comment:
- [ ] Good work building the featured speakers section. I have noticed a few things missing on it which I will highlight below in the diagram
 - The divider below the section title is missing in both deskotp & mobilr version ⚠️ 
 - The images are much larger than they should actually be in the mobile version ⚠️ 
 **Design**
 
![speakers](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/99656687/af738f9c-bcdc-4c68-bd45-ef426e4830c4)

**Your design**
![speakers 2](https://github.com/MustafaSamiShirzad/Capstone-Project/assets/99656687/787b872d-8724-4f84-9023-03344c58095d)

Reply:
Resolved

Reply:
- [ ] As suggested by the previous reviewer, Kindly reduce the size of the speaker's image to make more space for the text on the right side to follow Figma's design

> * [x]  Good work building the featured speakers section. I have noticed a few things missing on it which I will highlight below in the diagram
> * The divider below the section title is missing in both deskotp & mobilr version ⚠️
> * The images are much larger than they should actually be in the mobile version ⚠️
>   **Design**
> 
> ![speakers](https://user-images.githubusercontent.com/99656687/249114486-af738f9c-bcdc-4c68-bd45-ef426e4830c4.png)
> 
> **Your design** ![speakers 2](https://user-images.githubusercontent.com/99656687/249114598-787b872d-8724-4f84-9023-03344c58095d.png)